bytes/bytes_test.ts

```diff
@@ -9,15 +9,15 @@ import { assertEquals } from "../testing/asserts.ts";
 
 test(function bytesBytesFindIndex(): void {
   const i = bytesFindIndex(
-    new Uint8Array([1, 2, 0, 1, 2, 0, 1, 2, 0, 1, 3]),
+    new Uint8Array([0, 1, 0, 1, 2, 0, 1, 2, 0, 1, 3]),
     new Uint8Array([0, 1, 2])
   );
   assertEquals(i, 2);
 });
 
 test(function bytesBytesFindLastIndex1(): void {
   const i = bytesFindLastIndex(
-    new Uint8Array([0, 1, 2, 0, 1, 2, 0, 1, 3]),
+    new Uint8Array([0, 1, 2, 0, 1, 2, 1, 2]),
```

Comment:
Rather than changing the existing tests, can you add new one that demonstrates the failure?

Reply:
All right, done with 6abef85 and 17679af.